js/src/helpers/addPreview.js

```diff
@@ -0,0 +1,24 @@
+/**
+ * External dependencies
+ */
+import * as React from 'react';
+
+/**
+ * Internal dependencies
+ */
+import * as previewComponents from '../components';
```

Comment:
This could instead import from the common component package we talked about:

```js
import * as blockComponents from '@foo-vendor/package-name';
```

Reply:
What do you think about this convention for block names mapped to component names?

| Block name (without namespace)    | React Component name    |
| :------------- | :----------: |
|  `example-block` | `ExampleBlock`   | 
| `event-longer-block-name` | `EvenLongerBlockName`  | 

In GCB, block names are usually in a dash format, like `foo-bar`. It doesn't allow snake_case or camelCase.

Of course, this project should work also without GCB.

Reply:
Now outdated